cross/arp-scan/patches/001-update-get-iab.patch

```diff
@@ -0,0 +1,23 @@
+# update url for iab.txt
+# fix wide character output
```

Comment:
Please do not "hesitate" to submit such patch upstream to ease future maintenance/update

Reply:
@ymartin59 just found that this works with the current release